Greenstand/treetracker-android
# 417 Planter account payment data integration

This change includes the api integration (yet to be developed) to retrieve payment data for the planter account and saves it to the local database using the planter identifier. The planter identifier is either the email or phone number associated with the planter.  Each sync invokes the planter account api for all the planters on the device and caches the result on a local db. 

Note: Currently the dev build.gradle points to a locally running nodejs server with a mock implementation for the planter account api and this has to be replaced with the real one when it becomes available.

Test:
1. Install a fresh greenstand treetracker dev variant after removing any local instances already running using ` .\adb uninstall org.greenstand.android.TreeTracker.dev`
2. The migration included will work for any prior release version of the app and thus uninstalling app cleanly is not necessary in the production release version.
3. Run the app and go to tree data fragment and click Sync button. You will see the payment and tree data updated assuming if you have a fake api running locally in your network where the app is running.
